{
  "name": "napari-error-reporter",
  "id": "62a723cbaae6eba13a8cee31",
  "package_types": [
    "conda"
  ],
  "summary": "Opt-in automated bug/error reporting for napari",
  "description": "",
  "home": "https://github.com/tlambert03/napari-error-reporter",
  "public": true,
  "owner": {
    "company": "",
    "location": "",
    "login": "conda-forge",
    "name": "conda-forge",
    "url": "",
    "description": "A community-led collection of recipes, build infrastructure, and distributions for the conda package manager.",
    "created_at": "2015-04-11 10:15:08.727000+00:00",
    "user_type": "org"
  },
  "full_name": "conda-forge/napari-error-reporter",
  "url": "http://api.anaconda.org/packages/conda-forge/napari-error-reporter",
  "html_url": "http://anaconda.org/conda-forge/napari-error-reporter",
  "versions": [
    "0.3.0",
    "0.3.1"
  ],
  "latest_version": "0.3.1",
  "conda_platforms": [
    "noarch"
  ],
  "revision": 4,
  "license": "BSD-3-Clause",
  "license_url": null,
  "dev_url": null,
  "doc_url": null,
  "source_git_url": null,
  "source_git_tag": null,
  "app_entry": {},
  "app_type": {},
  "app_summary": {},
  "builds": [
    "pyhd8ed1ab_0"
  ],
  "releases": [
    {
      "version": "0.3.0",
      "distributions": [],
      "full_name": "conda-forge/napari-error-reporter/0.3.0"
    },
    {
      "version": "0.3.1",
      "distributions": [],
      "full_name": "conda-forge/napari-error-reporter/0.3.1"
    }
  ],
  "watchers": 1,
  "upvoted": 0,
  "created_at": "2022-06-13 11:47:20.647000+00:00",
  "modified_at": "2023-06-18 08:41:46.411000+00:00",
  "files": [
    {
      "description": null,
      "distribution_type": "conda",
      "basename": "noarch/napari-error-reporter-0.3.0-pyhd8ed1ab_0.tar.bz2",
      "attrs": {
        "build_number": 0,
        "license": "BSD-3-Clause",
        "has_prefix": false,
        "noarch": "python",
        "machine": null,
        "platform": null,
        "depends": [
          "appdirs",
          "python >=3.8",
          "qtpy",
          "sentry-sdk"
        ],
        "build": "pyhd8ed1ab_0",
        "timestamp": 1655120807708,
        "arch": null,
        "operatingsystem": null,
        "target-triplet": "None-any-None",
        "subdir": "noarch",
        "constrains": []
      },
      "upload_time": "2022-06-13 11:47:21.139000+00:00",
      "md5": "0d18c2b377c9b11cb765d6d77945485d",
      "sha256": "None",
      "size": 16769,
      "full_name": "conda-forge/napari-error-reporter/0.3.0/noarch/napari-error-reporter-0.3.0-pyhd8ed1ab_0.tar.bz2",
      "download_url": "//api.anaconda.org/download/conda-forge/napari-error-reporter/0.3.0/noarch/napari-error-reporter-0.3.0-pyhd8ed1ab_0.tar.bz2",
      "type": "conda",
      "version": "0.3.0",
      "owner": "conda-forge",
      "labels": [
        "main"
      ]
    },
    {
      "description": null,
      "distribution_type": "conda",
      "basename": "noarch/napari-error-reporter-0.3.1-pyhd8ed1ab_0.tar.bz2",
      "attrs": {
        "build_number": 0,
        "license": "BSD-3-Clause",
        "has_prefix": false,
        "noarch": "python",
        "machine": null,
        "platform": null,
        "depends": [
          "appdirs",
          "python >=3.8",
          "qtpy",
          "sentry-sdk"
        ],
        "build": "pyhd8ed1ab_0",
        "timestamp": 1655890753335,
        "arch": null,
        "operatingsystem": null,
        "target-triplet": "None-any-None",
        "subdir": "noarch",
        "constrains": []
      },
      "upload_time": "2022-06-22 09:39:39.050000+00:00",
      "md5": "73ad0a0149401b4cb13bff1c734a87c0",
      "sha256": "None",
      "size": 17641,
      "full_name": "conda-forge/napari-error-reporter/0.3.1/noarch/napari-error-reporter-0.3.1-pyhd8ed1ab_0.tar.bz2",
      "download_url": "//api.anaconda.org/download/conda-forge/napari-error-reporter/0.3.1/noarch/napari-error-reporter-0.3.1-pyhd8ed1ab_0.tar.bz2",
      "type": "conda",
      "version": "0.3.1",
      "owner": "conda-forge",
      "labels": [
        "main"
      ]
    }
  ]
}